Main.lean

```diff
@@ -0,0 +1,3 @@
+import LSpec.Runner
+
+def main : IO UInt32 := runner
```

Comment:
Can we just explicitly put the runner code here?

Reply:
Yeah feel free to put it here.